css/index.css

```diff
@@ -94,24 +116,109 @@ p {
     padding: 0 2%;
     margin-bottom: 0;
 }
-
-.middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
-}
-
-.middle-content .boxes .box {
-    width: 12.5%;
-    height: 100px;
-    background: black;
-    margin: 20px 2.5%;
-    color: white;
+h2 {
     display: flex;
     align-items: center;
-    justify-content: center;
+    justify-content: flex-start;
+}
+.boxes{display:flex;
+       flex-direction:row;
+       justify-content:center;
+}
```

Comment:
The parent style (.boxes) will cascade down to the children.  The parent contains all the styling the child needs except color.